index.html

```diff
@@ -294,9 +294,10 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 		                    <img class="col-sm-2" alt="Aleksandr Lymarenko" src="img/a-v-lymarenko.png">
 		                    <img class="col-sm-2" alt="Naveen Rajan" src="img/naveen.jpg">
 		                    <img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
+                            <img class="col-sm-2" alt="Reza Juliandri" src="img/reza_juliandri.jpg">
```

Comment:
Intending these lines with 7 more lines (to be exact) would be great I guess. Can you do that pls?

Reply:
please wait my git result error xd

Reply:
@ppabcd don't rush take your own time :)

Reply:
done :)

Reply:
I was referring to the above particular line.